lru.go

```diff
@@ -15,6 +15,7 @@ type Cache struct {
 	evictList *list.List
 	items     map[interface{}]*list.Element
 	lock      sync.RWMutex
+	OnEvicted func(key interface{}, value interface{})
```

Comment:
Fair enough - I just modeled it on the groupcache LRU implentation.  I'll make another pass that doesn't expose the internals.